lib/src/supabase_auth.dart

```diff
@@ -178,7 +178,7 @@ class SupabaseAuth with WidgetsBindingObserver {
   /// if _authCallbackUrlHost not init, we treat all deeplink as auth callback
   bool _isAuthCallbackDeeplink(Uri uri) {
     if (_authCallbackUrlHostname == null) {
-      return uri.queryParameters.containsKey('access_token');
+      return uri.toString().contains('access_token');
```

Comment:
Instead of searching the whole URI, I would just check the `uri.fragment`. That should be the part after #

Reply:
Sounds good.
```suggestion
      return uri.fragment.contains('access_token');
```